libcontainer/setns_init_linux.go

```diff
@@ -33,11 +34,18 @@ func (l *linuxSetnsInit) Init() error {
 	runtime.LockOSThread()
 	defer runtime.UnlockOSThread()
 
+	// If config.json is modified after the container started, we should validate selinux again.
+	if l.config.ProcessLabel != "" && !selinux.GetEnabled() {
+		return fmt.Errorf("selinux label is specified in config, but selinux is disabled or not supported")
+	}
```

Comment:
From memory, the config.json is not re-parsed for `runc exec` -- we use the config that's in `/run/runc/$ctr/...` which means that there's no chance it could change between runs (other than `runc update` but that only really does `cgroup` changes).

Reply:
> From memory, the config.json is not re-parsed for `runc exec`

No, it re-parsed in here: https://github.com/opencontainers/runc/blob/84cba4cbc1767353ae9d999c61f62a50c538589f/exec.go#L170

Reply:
@cyphar Besides `re-parse` config.json bundle dir, there are 3 situations use selinux in `runc exec`, I have update the PR description.
(1) use '--process-label' in runc exec;
(2) add selinuxLabel in process.json when use --process in runc exec;
(3) add selinuxLabel to config.json after the container started.

Reply:
I move the `selinux` validate to `utils_linux.go` when `exec`, so we can do this check before we start `runc init`. Does it make sense?

Reply:
Yes, it would be much better to do it *a lot* earlier.